MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,572 @@
+};
+
+ImageEditorToolbar annotationToolbar = new ();
+editorToolbar1.ToolbarItems = new List<IImageEditorToolbarItem>()
+{
+    new ImageEditorToolbarItem(){ Name = "Text"},
+    new ImageEditorToolbarItem(){ Name = "Shape"},
+    new ImageEditorToolbarItem(){ Name = "Pen"}
+};
+
+imageEditor.Toolbars.Add(editorToolbar);
+imageEditor.Toolbars.Add(annotationToolbar);
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add new custom toolbar 
+
+Grouping toolbar items allows you to organize actions efficiently. You can place items in various alignments to suit your app's design.
+
+{% tabs %}
+
+{% highlight XAML %}
+
+<imageEditor:SfImageEditor Source="image.png"
+                           AutoGenerateToolbarItems="False">
+    <imageEditor:SfImageEditor.Toolbars>
+        <imageEditor:ImageEditorToolbar Position="End">
+            <imageEditor:ImageEditorToolbar.ToolbarItems>
+                <imageEditor:ImageEditorToolbarGroupItem Alignment="Start">
+                    <imageEditor:ImageEditorToolbarGroupItem.Items>
+                        <imageEditor:ImageEditorToolbarItem Name="Browse" />
+                        <imageEditor:ImageEditorToolbarItem Name="Save" />
+                    </imageEditor:ImageEditorToolbarGroupItem.Items>
+                </imageEditor:ImageEditorToolbarGroupItem>
+                <imageEditor:ImageEditorToolbarGroupItem Alignment="End">
+                    <imageEditor:ImageEditorToolbarGroupItem.Items>
+                        <imageEditor:ImageEditorToolbarItem Name="Reset" />
+                        <imageEditor:ImageEditorToolbarItem Name="SaveEdit" />
+                    </imageEditor:ImageEditorToolbarGroupItem.Items>
+                </imageEditor:ImageEditorToolbarGroupItem>
+            </imageEditor:ImageEditorToolbar.ToolbarItems>
+        </imageEditor:ImageEditorToolbar>
+    </imageEditor:SfImageEditor.Toolbars>
+</imageEditor:SfImageEditor>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+imageEditor.AutoGenerateToolbarItems = false;
+ImageEditorToolbar editorToolbar = new ();
+editorToolbar.Position = ToolbarPosition.End;
+
+ImageEditorToolbarGroupItem browseGroup = new ()
+{
+    Alignment = ToolbarGroupItemAlignment.Start,
+    Items = new List<ImageEditorToolbarItem>()
+        {
+            new ImageEditorToolbarItem(){ Name ="Browse"},
+            new ImageEditorToolbarItem(){ Name ="Reset"  }
+        }
+};
+
+ImageEditorToolbarGroupItem saveGroup = new ()
+{
+    Alignment = ToolbarGroupItemAlignment.End,
+    Items = new List<ImageEditorToolbarItem>()
+        {
+            new ImageEditorToolbarItem(){ Name ="Save"},
+            new ImageEditorToolbarItem(){ Name ="SaveEdit"}
+        }
+};
+
+editorToolbar.ToolbarItems.Add(browseGroup);
+editorToolbar.ToolbarItems.Add(saveGroup);
+imageEditor.Toolbars.Add(editorToolbar);
+    
+{% endhighlight %}
+
+{% endtabs %}
+
```

Comment:
Add topic

## SubToolbar with explicit toolbars

Add example to how to add subtoolbars

Reply:
Added topic and example